common/darwin/Classes/FlutterRTCMediaStream.m

```diff
@@ -321,6 +321,58 @@ - (int)getConstrainInt:(NSDictionary*)constraints forKey:(NSString*)key {
   return 0;
 }
 
+- (RTCMediaStreamTrack*)cloneTrack:(nonnull NSString*)trackId {
+    NSString* newTrackId = [[NSUUID UUID] UUIDString];
+
+    RTCMediaStreamTrack *originalTrack = [self trackForId:trackId peerConnectionId: nil];
+    LocalVideoTrack* originalLocalTrack = self.localTracks[trackId];
+
+    if ([originalTrack.kind isEqualToString:@"audio"]) {
+      RTCVideoTrack *originalAudioTrack = (RTCAudioTrack *)originalTrack;
+
+      RTCAudioTrack* audioTrack = [self.peerConnectionFactory audioTrackWithTrackId:trackId];
+      LocalAudioTrack *localAudioTrack = [[LocalAudioTrack alloc] initWithTrack:audioTrack];
+
+      audioTrack.settings = originalAudioTrack.settings;
+      [self.localTracks setObject:localAudioTrack forKey:newTrackId];
+
+      for (NSString* streamId in self.localStreams) {
+        RTCMediaStream* stream = [self.localStreams objectForKey:streamId];
+        for (RTCAudioTrack* track in stream.audioTracks) {
+          if ([trackId isEqualToString:track.trackId]) {
+            [stream addAudioTrack:audioTrack];
+          }
+        }
+      }
+
+      return audioTrack;
+    } else if ([originalTrack.kind isEqualToString:@"video"]) {
```

Comment:
```suggestion
    } else if (originalTrack != nil && [originalTrack.kind isEqualToString:@"video"]) {
```
If the `originalTrack` is `nil` for any reason, accessing `kind` will cause a crash.